tensorboard/webapp/runs/views/runs_table/runs_group_menu_button.ng.html

```diff
@@ -0,0 +1,68 @@
+<!--
```

Comment:
Change the file name to `runs_group_menu_button_component.ng.html` to mirror that of TypeScript component file.

Reply:
Thanks for the catch. Updated.